src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -41,11 +42,13 @@ export const EDU_PAYMENT_INFORMATION_SAVE_SUCCEEDED =
 export const EDU_PAYMENT_INFORMATION_SAVE_FAILED =
   'EDU_PAYMENT_INFORMATION_SAVE_FAILED';
 
+const isProd = () => (!environment.isProduction() ? 'cnp-' : '');
+
 export function fetchCNPPaymentInformation(recordEvent = recordAnalyticsEvent) {
   return async dispatch => {
     dispatch({ type: CNP_PAYMENT_INFORMATION_FETCH_STARTED });
 
-    recordEvent({ event: 'profile-get-direct-deposit-started' });
+    recordEvent({ event: `profile-get-${isProd}direct-deposit-started` });
```

Comment:
need to actually call `isProd` here

Reply:
good catch!

Reply:
there actually was no need to have `isProd` be a function instead of just a const 🤷‍♀️